app/menus/members/menu.rb

```diff
@@ -24,50 +24,49 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #
 # See COPYRIGHT and LICENSE files for more details.
-#++
+# ++
+module Members
+  class Menu < Submenu
+    attr_reader :project, :params
 
-module Menus
-  module MembersHelper
-    def first_level_menu_items
-      [OpenProject::Menu::MenuGroup.new(header: nil, children: user_status_options)] + nested_menu_items
-    end
-
-    private
+    def initialize(project: nil, params: nil)
+      @project = project
+      @params = params
```

Comment:
These two lines are not needed. `Submenu` already sets `@project` and `@params`.

Reply:
I fixed it in a commit.